app/views/rating_questions/show.html.erb

```diff
@@ -0,0 +1,12 @@
+
+<div class="form">
+  <div class="flash-notice">
```

Comment:
Move this flash notice to the application layout.

Reply:
done